plugin/metrics/prometheus/metricsstore/reader_test.go

```diff
@@ -61,22 +64,36 @@ var defaultConfig = config.Configuration{
 	LatencyUnit:                 "ms",
 }
 
+func createTracingProvider() trace.TracerProvider {
+	exporter, err := stdout.New(stdout.WithPrettyPrint())
+	if err != nil {
+		panic(err)
+	}
+	tp := sdktrace.NewTracerProvider(
+		sdktrace.WithSampler(sdktrace.AlwaysSample()),
+		sdktrace.WithBatcher(exporter),
+	)
+	return tp
```

Comment:
```suggestion
	return tp, exporter
```

Reply:
Sorry! I couldn't get that. Why `exporter`?

Reply:
because that allows you to do `exporter.GetSpans()` and validate that the correct spans were created

Reply:
@yurishkuro while testing out I am not getting spans to validate that the correct ones were created
`assert.NotEmpty(t, exp.GetSpans(), "No spans recorded during the test.")`
```
Error:          Should NOT be empty, but was []
                Test:           TestInvalidCertFile
                Messages:       No spans recorded during the test.
```
LMK if I have missed somewhere